cypress/support/commands.js

```diff
@@ -22,4 +22,14 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('getElementById', (id) => {
+  cy.get(`#${id}`);
+});
+
+// Cypress.Commands.add('assertAlert', (alertText) => {
+//   cy.on('window:alert', (str) => {
+//     expect(str).to.equal(alertText);
+//   });
+// });
```

Comment:
Remove comments everywhere

Reply:
Fixed. By the way this command didn't work for some reason.